source/extensions/extensions_build_config.bzl

```diff
@@ -269,6 +269,14 @@ EXTENSIONS = {
     #
 
     "envoy.http.stateful_header_formatters.preserve_case":       "//source/extensions/http/header_formatters/preserve_case:preserve_case_formatter",
+
+    #
+    # Original IP detection
+    #
+
+    "envoy.http.original_ip_detection.custom_header":        "//source/extensions/original_ip_detection/custom_header:config",
```

Comment:
optional nit, I'd be inclined to say this one doesn't need to be compiled in by default.

Reply:
I'd agree, I forget if I pulled it in for testing or something. I'll kick it back out.

Reply:
yeah, you should be able to link explicitly in tests.  If you make it default out,  I think it's worth a comment line as we do for default-out http extensions.

Reply:
Ok, I remember what I left this one in: it's convenient for testing in test/extensions/filters/network/http_connection_manager/config_test.cc when we want to test something else besides the default extension (XFF).

I could get rid of it and add a toy extension for testing, but I think given it's pretty small we probably can/should keep it around... Your call though.

Reply:
ah, you should be able to just direct link the extension into the config test.  Arguably that's not great w.r.t the extension policy, but it's still better than having it linked in everyone's production binaries.